ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -137,14 +138,14 @@ export const hasExpired = (expiration: number): boolean =>
   !!expiration && getWholeDaysFromNow(expiration) < 0;
 
 // The module can either be expired or is expiring
-export const isExpiring = (expiration?: number): boolean =>
```

Comment:
This was a mistake.  The param is not *optional* but it may be *undefined*, which is still handled.